lib/core/src/display/shape/text/content.rs

```diff
@@ -1,28 +1,33 @@
-#![allow(missing_docs)]
+//! Module with all structures describing the content of the TextField.
 pub mod line;
 
 use crate::prelude::*;
 
-use crate::display::shape::text::font::FontId;
-use crate::display::shape::text::font::FontRenderInfo;
-use crate::display::shape::text::font::Fonts;
+use crate::display::shape::glyph::font::FontId;
+use crate::display::shape::glyph::font::FontRegistry;
+use crate::display::shape::glyph::font::FontRenderInfo;
 use crate::display::shape::text::content::line::Line;
-use crate::display::shape::text::content::line::LineRef;
+use crate::display::shape::text::content::line::LineFullInfo;
+use crate::display::shape::text::TextFieldProperties;
 
+use nalgebra::Vector2;
 use std::ops::Range;
 use std::ops::RangeFrom;
 use std::ops::RangeInclusive;
 
 
+
 // ==================
 // === DirtyLines ===
 // ==================
 
 /// Set of dirty lines' indices
 #[derive(Debug)]
 pub struct DirtyLines {
-    pub single_lines : HashSet<usize>,
-    pub range        : Option<RangeFrom<usize>>
+    /// A set of single edited lines.
+    pub single_lines: HashSet<usize>,
+    /// An open range of dirty lines. When the line is added or removed, all the lines below.
```

Comment:
I dont understand this sentence:
```When the line is added or removed, all the lines below.```

Reply:
certainly, I've eaten some words from it

Reply:
omnomnom :D